GitUp/Application/AppDelegate.m

```diff
@@ -299,6 +263,36 @@ - (void)handleDocumentCountChanged {
   }
 }
 
+#pragma mark - Loading Xibs
+- (id)_loadWindowFromBundleXibWithName:(NSString *)name expectedClass:(Class)class {
+  NSBundle *mainBundle = [NSBundle mainBundle];
+  NSArray *objects = @[];
+  [mainBundle loadNibNamed:name owner:self topLevelObjects:&objects];
+  NSArray *filteredObjects = [objects filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(id  _Nullable evaluatedObject, NSDictionary<NSString *,id> * _Nullable bindings) {
+    return [evaluatedObject isKindOfClass:class];
+  }]];
+  return filteredObjects.firstObject;
+}
```

Comment:
Do we expect to use this for other classes. My ideal would be that we just have a WelcomeWindow init method that does all the xib loading work for us. Could we perhaps move this logic into that class? Other than that, I think this PR looks pretty good.

Reply:
@lucasderraugh 
In previous PR I load windows and About panel by this method.
Also, look at `-loadNibNamed` invocation. It specifies owner as `self` ( AppDelegate ). Other windows don't have too much incapsulated logic ( I leave them `as-is` with some methods in `AppDelegate` ).